packages/db/package.json

```diff
@@ -6,6 +6,7 @@
   "module": "./dist/prismaClient.mjs",
   "types": "./dist/prismaClient.d.ts",
   "scripts": {
+    "build": "tsup prisma/prismaClient.ts --format esm,cjs",
```

Comment:
does prisma need to be built on its own? it will be bundled by the app using it 🤔